test/bigtest/network/factories/exportConfig.js

```diff
@@ -0,0 +1,10 @@
+import { Factory } from 'miragejs';
+import faker from 'faker';
+
+export default Factory.extend({
+  id: faker.random.uuid,
+  metadata: {
```

Comment:
this factory doesn't look useful, usually we try to put as many fields as possible, to get test data similar to the back-end

Reply:
still active